src/antelope/stores/history.ts

```diff
@@ -272,8 +271,8 @@ export const useHistoryStore = defineStore(store_name, {
                 const systemTokenDecimals = chainSettings.getSystemToken().decimals;
 
                 // all contracts in transactions are already cached, no need to use getContract
-                const toPrettyName = contractStore.__cachedContracts[tx.to?.toLowerCase()]?.name ?? '';
-                const fromPrettyName = contractStore.__cachedContracts[tx.from?.toLowerCase()]?.name ?? '';
+                const toPrettyName = contractStore.getContractIfStored(label, tx.to?.toLowerCase())?.name ?? '';
```

Comment:
there is an issue here where tx.to / tx.from may be undefined. in this case, `getContractIfStored` throws an error because it has the line `const addressLower = address.toLowerCase();`

i think it should be refactored like
```
const toPrettyName = contractStore.getContractIfStored(label, tx.to?.toLowerCase() ?? '')?.name ?? '';
const fromPrettyName = contractStore.getContractIfStored(label, tx.from?.toLowerCase() ?? '')?.name ?? '';
```
this gets rid of the error locally for me

Reply:
correct